### PR TITLE
Added function for writing extra known tokens

### PR DESCRIPTION
**Pull request by mwwolters**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/ok-to-test
/priority backlog

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -446,6 +446,19 @@ function append_or_replace_prefixed_line {
...
+# write_extra_known_tokens assumes that EXTRA_KNOWN_TOKENS is formatted like:
+# $token1,system:$user1,uid:$user1 $token2...
+#
+# $1: is the known tokens file
+#
+# Assumes vars (supplied via kube-env):
+# EXTRA_KNOWN_TOKENS
+function write_extra_known_tokens {
+  for line in ${EXTRA_KNOWN_TOKENS[@]}; do
+    append_or_replace_prefixed_line $1 line
```
Comment (tallclair) [inline cluster/gce/gci/configure-helper.sh]:
```suggestion
    append_or_replace_prefixed_line "$1" "$line"
```

**Pull request closed without merging**